plugins/woocommerce/e2e/tests/admin-analytics/analytics-overview.spec.js

```diff
@@ -1,19 +1,54 @@
 const { test, expect } = require( '@playwright/test' );
 
 test.describe( 'Analytics pages', () => {
-	test.use( { storageState: 'e2e/storage/adminState.json' } );
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.afterEach( async ( { page } ) => {
+		// do some cleanup after each test to make sure things are where they should be
+		await page.goto(
+			'wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview',
+			{ waitForLoadState: 'networkidle' }
+		);
+
+		// Grab all of the section headings
+		const sections = await page.$$(
+			'h2.woocommerce-section-header__title'
+		);
+		if ( sections.length < 3 ) {
+			// performance section is hidden
+			await page.click( '//button[@title="Add more sections"]' );
+			await page.click( '//button[@title="Add Performance section"]' );
+			await page.waitForSelector( 'h2:has-text("Performance")', {
+				state: 'visible',
+			} );
+			await page.waitForLoadState( 'networkidle' );
+		}
+		const lastSection = await page.textContent(
+			'h2.woocommerce-section-header__title >> nth=2'
+		);
+		if ( lastSection === 'Performance' ) {
+			// sections are in the wrong order
+			await page.click(
+				'//button[@title="Choose which analytics to display and the section name"]'
+			);
+			await page.click( 'text=Move up' );
+			await page.click(
+				'//button[@title="Choose which analytics to display and the section name"]'
+			);
+			await page.click( 'text=Move up' );
+			await page.waitForTimeout( 1000 );
+		}
+	} );
```

Comment:
I moved all of the cleanup steps out to the afterEach hook.  That way they'll be run regardless of whether the test passes or fails -- seems to improve reliability.